onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -330,6 +330,11 @@ class PlannerImpl {
     for (auto it = freelist_.begin(); it != freelist_.end(); ++it) {
       size_t reusable = static_cast<size_t>(it->ml_value);
       const onnxruntime::NodeArg* p_node_arg = ort_value_info_.at(reusable).p_def_site;
+      if (!p_node_arg) {
+        // TODO this should be an error case, needs more investigation
+        // https://msdata.visualstudio.com/Vienna/_workitems/edit/724826/
+        continue;
```

Comment:
should we be checking in links to internal work items in the public github?

Reply:
i think it's ok - i like the searchability of it. is there a good reason not to?

---
In reply to: [414367443](https://github.com/microsoft/onnxruntime/pull/3681#discussion_r414367443) [](ancestors = 414367443)